### PR TITLE
Add Red Sky Ops

### PR DESCRIPTION
**Pull request by jgustie**

### CODE REVIEW

File: charts/redskyops/v0.1.1/questions.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+questions:
```
Comment (guangbochen) [inline charts/redskyops/v0.1.1/questions.yml]: can u please add those labels to the top of questions.yml:
```
labels:
  io.cattle.role: cluster # options are cluster/project
```

File: charts/redskyops/v0.1.1/templates/crds.yaml [hunk 2]
```diff
@@ -0,0 +1,13473 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```
[the remark this answers is not in the document]
Comment (jgustie) [reply charts/redskyops/v0.1.1/templates/crds.yaml]: The chart itself does not include any resource definitions for the types being introduced so the hook was intentionally omitted as suggested by the Helm documentation, is this going to be a problem for the Rancher chart?

[the remark this answers is not in the document]
Comment (guangbochen) [reply charts/redskyops/v0.1.1/templates/crds.yaml]: This is ok if we don't have pre-request of CRD resources, thanks.

**Comment by guangbochen** [comment]
LGTM and merged.

**Pull request merged**